lib/dradis/plugins/content_service/boards.rb

```diff
@@ -3,12 +3,13 @@ module Boards
     extend ActiveSupport::Concern
 
     def all_boards
-      Board.where(project_id: project.id)
+      project.methodology_library.boards
     end
 
     def create_board(args={})
-      name = args.fetch(:name, default_board_name)
-      Board.create(name: name, project_id: project.id)
+      name    = args.fetch(:name, default_board_name)
+      node_id = args.fetch(:node_id, project.methodology_library.id)
```

Comment:
@aapomm why do you use a private method to set the default in L10, but set the default directly, without a private method in L11?

Reply:
Good point. Using a method is consistent with the other services. Updated!